script/openqa-clone-custom-git-refspec

```diff
@@ -55,6 +53,23 @@ extract_urls_from_pr() {
     fi
 }
 
+config_file="$HOME/.config/openqa/github-token.conf"
+get_token_from_file() {
+    token=$(grep "^GITHUB_TOKEN=" "$config_file" | cut -d'"' -f2)
+    echo "$token"
+}
+
+if [[ -n "$GITHUB_TOKEN" ]]; then
+    AUTHENTICATED_REQUEST=" -u $GITHUB_TOKEN:x-oauth-basic"
+    echo "Github oauth token provided in environment variable, performing authenticated requests."
+elif [[ -f $config_file ]]; then
+    AUTHENTICATED_REQUEST=" -u $(get_token_from_file):x-oauth-basic"
+    echo "GitHub oauth token found in config file, performing authenticated requests."
```

Comment:
```suggestion
    echo "GitHub OAuth token found in config file, performing authenticated requests."
```

Reply:
Afaik this is the official spelling of OAuth.
https://en.m.wikipedia.org/wiki/OAuth